src/ensembl/src/content/app/browser/drawer/drawer-views/DrawerTranscript.tsx

```diff
@@ -12,7 +12,7 @@ type DrawerTranscriptProps = {
   ensObject: EnsObject;
 };
 
-const TRANSCRIPT_GENE_NAME = 'gene-feat-1';
+const TRANSCRIPT_GENE_NAME = 'track:gene-feat-1';
```

Comment:
We should now be using constants like these for the track names. Can we move this to a file that has all the track name constants?

Reply:
Added a comments saying that it'll be moved or removed when we support multiple transcripts.